SQL/updates/36-37.py

```diff
@@ -0,0 +1,153 @@
+            print("Book {} named \"{}\" is programattic: removing record...".format(entry[0], entry[1]))
+            programmatic_books += 1
+            remove_rows.append(entry)
+            new_rows.remove(entry)
+            continue
+        if len(entry[3]) < 150:
+            print("Book {} is less than 150 characters: removing record...".format(entry[0]))
+            short_books += 1
+            remove_rows.append(entry)
+            new_rows.remove(entry)
+            continue
+        for book in new_rows:
+            if entry[1] == book[1] and entry != book:
+                if entry[3] == book[3]:
+                    print("Book {} and Book {} have the same title and content: removing record {}...".format(entry[0],book[0],book[0]))
+                    remove_rows.append(book)
+                    new_rows.remove(book)
+                    duplicate_books += 1
+        #here we're turning our content string into a JSON list
+    print("All Books vetted, books marked for deletion report:")
+    print("Duplicate Books:{}".format(duplicate_books))
+    print("No Title Books:{}".format(notitle_books))
+    print("Programmatic Books:{}".format(programmatic_books))
+    print("Short Books:{}".format(short_books))
+    print("Total Books Marked for Deletion: {}".format(len(remove_rows)))
+    print("Generated {} rows to insert into new table".format(len(new_rows)))
+
+    if len(new_rows) == 0:
+        print("ERROR: No rows have been modfied, no update will be commited")
+
+
+    print("Inserting...")
+    for row in new_rows:
+        data = [row[2],row[3],row[0]]
+        sql_query = "UPDATE library SET category=%s, content=%s WHERE id=%s"
+        cursor.execute(sql_query, data)
+
+
+    print("removing {} marked records...".format(len(remove_rows)))
+    for row in remove_rows:
+        data = [row[0]]
+        sql_query = "DELETE FROM library WHERE id=%s"
+        cursor.execute(sql_query, data)
+
+    print("Adding Summary Column")
+    cursor.execute("ALTER TABLE library ADD COLUMN summary MEDIUMTEXT DEFAULT ''")
+    print("Adding Ratings Column")
+    cursor.execute("ALTER TABLE library ADD COLUMN ratings DOUBLE DEFAULT 0")
+    cursor.execute("ALTER TABLE library ADD COLUMN raters MEDIUMTEXT DEFAULT ''")
+    print("Updating Flagged Column")
+    cursor.execute("ALTER TABLE library MODIFY COLUMN flagged MEDIUMTEXT")
+    cursor.execute("UPDATE library SET flagged=''")
+    cursor.execute("ALTER TABLE library RENAME COLUMN flagged TO reports")
+    print("Updating/Adding new Categories columns")
+    cursor.execute("ALTER TABLE library MODIFY COLUMN category INT")
+    cursor.execute("ALTER TABLE library RENAME COLUMN category to primaryCategory")
+    cursor.execute("ALTER TABLE library ALTER primaryCategory SET DEFAULT 0")
+    cursor.execute("ALTER TABLE library ADD COLUMN secondaryCategory INT DEFAULT 0")
+    cursor.execute("ALTER TABLE library ADD COLUMN tertiaryCategory INT DEFAULT 0")
+    cursor.close()
```

Comment:
These should be in a separate `.sql` file as seen in my comment above

Reply:
instead of making this python calls, this is taken care of in `37-38.sql` as part of the create table call